cmd/buildah/images.go

```diff
@@ -5,8 +5,14 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/urfave/cli"
+	"encoding/json"
 )
 
+type JsonImage struct {
+	Id string `json:"id"`
```

Comment:
`Id` should be `ID` (all uppercase) - you would also want an `omitempty` in case id isn't there? (`json:"id,omitempty"`)